frontend/src/commons/Header.js

```diff
@@ -24,12 +24,17 @@ export default function Header({ title, backbutton }) {
 }
 
 const HeaderStyled = styled.header`
+  position: fixed;
```

Comment:
I also set my header fixed at the beginning and got some problems through it. I then only used grid and  `overflow: scroll` in the list and it also stayed at the top. Only as a hint in case you get problems.